aspnetcore/blazor/common/samples/3.x/BlazorSample_Server/Pages/templated-components/Pets1.razor

```diff
@@ -0,0 +1,29 @@
+@page "/pets"
+
+<h1>Pets</h1>
+
+<TableTemplate Items="pets" Context="pet">
```

Comment:
Do we need the snippet comment here?

Reply:
It's only needed when the code is preceded with an `@code` block that isn't meant to be shown. Here for `Pet1`, it compiles and doesn't need a code block.